pkg/kubernetes/kubernetes.go

```diff
@@ -90,7 +93,10 @@ var ErrEmptyVersionTag = errors.New("got an empty version tag from Github")
 
 // Kubernetes is a client for Kubernetes.
 type Kubernetes struct {
-	client     client.KubeClientConnector
+	// client client.KubeClientConnector
+	client     *client.Client
```

Comment:
this `client` will be removed once all logic from `pkg/kubernetes/client/` + `pkg/kubernetes/client/customresources` layers is moved to `pkg/kubernetes/` layer